src/ShellCheck/Interface.hs

```diff
@@ -270,3 +274,36 @@ mockedSystemInterface files = SystemInterface {
             [] -> return $ Left "File not included in mock."
             [(_, contents)] -> return $ Right contents
 
+-- The Ranged class is used for types that has a start and end position.
```

Comment:
Btw I moved these typeclasses into `ShellCheck.Fixer` so that hopefully no one else will start depending on them